pylib/Tools/Launcher/SLURM.py

```diff
@@ -45,7 +45,18 @@
 class SLURM(LauncherMTTTool):
 
     def __init__(self):
+        # This covers a situation where the job is aborted but the slurm cleanup times out, leaving
+        # the job status as 0, here is the typical output
+        #     stderr: srun: Force Terminated job 6050
+        #     stderr: srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
+        #     stderr: srun: error: Timed out waiting for job step to complete
+        self.falsePass = {}
+        self.falsePass['errstr']  = "Timed out waiting for job step to complete"
+        self.falsePass['rtncode'] = 143
+        self.falsePass['result']  = 0     # testDef.MTT_TEST_FAILED
+
         # initialise parent class
+        # LauncherMTTTool.__init__(self, additionalCheck = self.falsePass)
```

Comment:
Did you leave this in intentionally?

Reply:
thanks, that's a testing artifact, the commented line is the one that should be uncommented ... none of these changes will take effect with out that line enabled.  Good catch.

Reply:
i've uploaded the fix